test/test.js

```diff
@@ -491,6 +491,19 @@ suite('Request', function() {
     })
   })
 
+  test('clone request bodies', function() {
+    var r1 = new Request('https://fetch.spec.whatwg.org/', {
+      method: 'post',
+      headers: {'content-type': 'text/plain'},
+      body: 'I work out'
```

Comment:
Do you think it would be useful to also add a test that verifies that Blob bodies can be cloned too? Take a look at other Blob test to see how to define such tests only for browsers that support Blob.